.circleci/src/commands/@web-commands.yml

```diff
@@ -93,6 +93,12 @@ web-distribute:
     - checkout
     - attach_workspace:
         at: ./
+    - run:
+        name: Install dependencies
+        command: rm -rf node_modules && SKIP_POD_INSTALL=true npm ci
```

Comment:
just curious, we couldn't use the cached version from the init phase?

Reply:
Actually a very crazy issue that I don't quite understand. Electron builder always fails doing symlinks from the node_gyp artifacts with a `Read Only File System` error. This is stuff that just exists in the node_modules folder. Only those node_gyp artifacts (like `python3` for example) throw this error when you try to modify them. But, making a copy of the file and renaming it to python3 fixes everything. Reinstalling node modules has the same effect. Maybe it has something to do with node_gyp compiling dependencies for one platform in the CI init step and then trying to consume them on a different platform (the mac os circle ci image). But idk how the change in dependency structure would have caused that

Reply:
I see what you mean about maybe compiling for different platforms but to your point it's weird that it worked before. Well something to note for monorepo